test/models/volunteer_state_test.rb

```diff
@@ -0,0 +1,209 @@
+require 'test_helper'
+
+class VolunteerStateTest < ActiveSupport::TestCase
+  def create_basic_accepted_volunteer
+    volunteer_user = create :user_volunteer, volunteer: create(:volunteer)
+    volunteer_user.volunteer
```

Comment:
What is this doing?

Reply:
returns the `Volunteer`, not the `User`

Reply:
@chrysalag but as you mention it, actually it isn't so nice. I made a sub factory of volunteer `:volunteer_with_user` that creates a volunteer, that has a volunteer role user with it.